### ISSUE TITLE
Update For `aria-describedby` in `<fieldset>` elements

### ISSUE THREAD

**Issue by andrewnordlund** [andrewnordlund/nordFormsMode#5]
As demonstrated on the [ARIA-Describedby and Labelledby Test Page](https://www.nordburg.ca/tests/ariaTest/labelledby-describedby.html) putting `aria-describedby` in a `<fieldset>` is pretty well supported.  So the "Partial Support" thingy should be replaced with the regular green dotted line.

**Comment by andrewnordlund** [comment]
Done.
